src/main/fc/tasks.c

```diff
@@ -481,7 +481,11 @@ task_t tasks[TASK_COUNT] = {
     [TASK_ACCEL] = DEFINE_TASK("ACC", NULL, NULL, taskUpdateAccelerometer, TASK_PERIOD_HZ(1000), TASK_PRIORITY_MEDIUM),
     [TASK_ATTITUDE] = DEFINE_TASK("ATTITUDE", NULL, NULL, imuUpdateAttitude, TASK_PERIOD_HZ(100), TASK_PRIORITY_MEDIUM),
 #endif
+#ifdef USE_RX_EXPRESSLRS
+    [TASK_RX] = DEFINE_TASK("RX", NULL, rxUpdateCheck, taskUpdateRxMain, TASK_PERIOD_HZ(33), TASK_PRIORITY_REALTIME),
```

Comment:
The only issue I see here is that if `USE_RX_EXPRESSLRS` is compiled in, but not actually used, then all RX systems use realtime priority.  

Can we set the task priority dynamically, based on the RX provider, and also check for realtime tx task in the scheduler?